cmd/bnsapi/handlers/handlers.go

```diff
@@ -563,7 +564,29 @@ func lastChunk(path string) string {
 }
 
 // DefaultHandler is used to handle the request that no other handler wants.
-type DefaultHandler struct{}
+type DefaultHandler struct {
+	Domain string
+}
+
+var availableEndpointsTempl = template.Must(template.New("").Parse(`
+Available endpoints with query parameter:
```

Comment:
`Available endpoints with query parameter` should be `Available endpoints with query parameters` (plural on parameters).